posts/month-in-mathlib-oct-2021.md

```diff
@@ -0,0 +1,66 @@
+
+* [PR #9462](https://github.com/leanprover-community/mathlib/pull/9462) :: feat(measure_theory/covering/besicovitch): the Besicovitch covering theorem
+* [PR #9461](https://github.com/leanprover-community/mathlib/pull/9461) :: feat(measure_theory/covering/besicovitch_vector_space): vector spaces satisfy the assumption of Besicovitch covering theorem
+* [PR #9576](https://github.com/leanprover-community/mathlib/pull/9576) :: feat(measure_theory/covering/besicovitch): the measurable Besicovitch covering theorem
+* [PR #9679](https://github.com/leanprover-community/mathlib/pull/9679) :: feat(measure_theory/covering/besicovitch): remove measurability assumptions
+* [PR #9680](https://github.com/leanprover-community/mathlib/pull/9680) :: feat(measure_theory/covering/vitali): Vitali covering theorems
+
+* [PR #8976](https://github.com/leanprover-community/mathlib/pull/8976) :: feat(order/jordan_holder): Jordan Hölder theorem
+
+* [PR #9252](https://github.com/leanprover-community/mathlib/pull/9252) :: feat(topology/homotopy): add `homotopy_with`
+* [PR #9141](https://github.com/leanprover-community/mathlib/pull/9141) :: feat(topology/homotopy/path): Add homotopy between paths
+
+* [PR #9521](https://github.com/leanprover-community/mathlib/pull/9521) :: feat(topology/algebra): adic topology
+* [PR #9589](https://github.com/leanprover-community/mathlib/pull/9589) :: feat(topology/algebra): valued fields
+
+* [PR #9646](https://github.com/leanprover-community/mathlib/pull/9646) :: feat(number_theory/liouville): Liouville numbers form a dense Gδ set
+* [PR #9702](https://github.com/leanprover-community/mathlib/pull/9702) :: feat(number_theory/liouville): the set of Liouville numbers has measure zero
+
+* [PR #9496](https://github.com/leanprover-community/mathlib/pull/9496) :: feat(analysis/box_integral): Divergence thm for a Henstock-style integral
+* [PR #9811](https://github.com/leanprover-community/mathlib/pull/9811) :: feat(measure_theory/integral): Divergence theorem for Bochner integral
+
+* [PR #9817](https://github.com/leanprover-community/mathlib/pull/9817) :: feat(field_theory/finite/galois_field): uniqueness of finite fields
+* [PR #9834](https://github.com/leanprover-community/mathlib/pull/9834) :: feat(field_theory/finite/galois_field): finite fields with the same cardinality are isomorphic
+
+* [PR #9540](https://github.com/leanprover-community/mathlib/pull/9540) :: feat(analysis/normed_space/banach): open mapping theorem for maps between affine spaces
+* [PR #9440](https://github.com/leanprover-community/mathlib/pull/9440) :: feat(analysis/asymptotics): Define superpolynomial decay of a function
+* [PR #9662](https://github.com/leanprover-community/mathlib/pull/9662) :: feat(group_theory/sylow): Frattini's Argument
+* [PR #7825](https://github.com/leanprover-community/mathlib/pull/7825) :: feat(combinatorics/hall): generalized Hall's Marriage Theorem
```

Comment:
Possible commentary:

Sometimes results about finite types can be lifted to ones about infinite types by a compactness argument (like via the compactness theorem in model theory), and Hall's marriage theorem is one such result.  Using the language of category theory, there is an inverse system of solutions to the marriage problem when restricted to finite subsets of the domain, and [the limit of this kind of inverse system is nonempty](https://leanprover-community.github.io/mathlib_docs/topology/category/Top/limits.html#nonempty_sections_of_fintype_inverse_system).

Reply:
Thanks. Done.